tests/sentry/workflow_engine/endpoints/test_validators.py

```diff
@@ -416,6 +416,13 @@ class MockDataSourceValidator(BaseDataSourceValidator[MockModel]):
     field2 = serializers.IntegerField()
     data_source_type_handler = QuerySubscriptionDataSourceHandler
 
+    class Meta:
+        model = MockModel
+        fields = [
+            "field1",
+            "field2",
+        ]
+
```

Comment:
are there any specific tests we should add to ensure the validator is working? thinking like a condition that isn't valid or a result that isn't valid. (or both)

Reply:
added some tests 👍